### PR TITLE
Fix all contributors badge

### PR DESCRIPTION
**Pull request by anku255**
### Changelog

1. Removed `<p class="github-only"></p` as it is no longer needed.
2. There were two badges for `all-contributors`. I have removed the extra badge.

### Test

I ran `all-contributors-cli` and tested addition of a new member. It works as expected.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
-<p class="github-only">
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-16-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
```
Comment (anku255) [inline readme.md:5]: I have removed this. Did we need it?

Comment (jevakallio) [reply readme.md:5]: Nope, this is ok. Not sure how this ended up here.

**Comment by anku255** [comment]
@jevakallio Take a look now 😃

**Comment by jevakallio** [comment]
@all-contributors, please add @anku255 and @sauravkhdoolia for doc contributions!

**Pull request merged**